<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Comparar arquivos</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="btnPasta">
    <property name="geometry">
     <rect>
      <x>437</x>
      <y>30</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Arquivos:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editPasta">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>421</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Pasta:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAbrir">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>290</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Abrir arquivos</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Colunas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPlotar">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>350</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Plotar</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Se houver arquivo .csv</string>
    </property>
    <widget class="QComboBox" name="comboSep">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Espaço</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>;</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>,</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboDecimal">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>.</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>,</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Decimal</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Separador</string>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="listArquivos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>381</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listColunas">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>381</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAtualizar">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>28</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Atualizar</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>290</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Salvar</string>
    </property>
    <widget class="QCheckBox" name="checkBoxCSV">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>58</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Salvar CSV</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxSVG">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>58</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Salvar SVG</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxPDF">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>28</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Salvar PDF</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxHTML">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>28</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Salvar HTML</string>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboOpcoes">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>320</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Linhas+Marcadores</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Linhas</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Marcadores</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>300</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Opções:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editFiltro">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>58</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
